sdk-commons/pom.xml

```diff
@@ -81,6 +86,11 @@
             <artifactId>mockito-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.json</groupId>
+            <artifactId>json</artifactId>
+            <version>20230618</version>
+        </dependency>
```

Comment:
Forgot this: we actually have Jackson as a dependency already. Can we convert the code to use that instead of adding a new JSON-handling library please?